### ISSUE TITLE
Bug with cookies authorization

### ISSUE THREAD

**Issue by romanleshukov** [daltoniam/Starscream#760]
Hi,

I have found a bug, which prevented establishing an authorized connection to server. We are using auth cookies. It is in the file: HTTPHandler.swift, method: createUpgrade, and the code: 
```swift
if let cookies = HTTPCookieStorage.shared.cookies(for: url), !cookies.isEmpty {
    let headers = HTTPCookie.requestHeaderFields(with: cookies)
    for (key, val) in headers {
        req.setValue(key, forHTTPHeaderField: val)
    }
}
```

It should be: 
```swift
req.setValue(val, forHTTPHeaderField: key)
```

Best regards,
Roman

**Comment by daltoniam** [comment]
Ah thanks. Total oversight on my part. Fixed in 4.0.3 (just released).

**Comment by romanleshukov** [comment]
Thank you!
